rules/sonic-ycable.mk

```diff
@@ -0,0 +1,19 @@
+# sonic-ycable (SONiC Y-Cable daemon) Debian package
+
+# SONIC_YCABLE_PY2 package
+
+SONIC_YCABLE_PY2 = sonic_ycable-1.0-py2-none-any.whl
+$(SONIC_YCABLE_PY2)_SRC_PATH = $(SRC_PATH)/sonic-platform-daemons/sonic-ycable
+$(SONIC_YCABLE_PY2)_DEPENDS = $(SONIC_PY_COMMON_PY2) $(SONIC_PLATFORM_COMMON_PY2)
+$(SONIC_YCABLE_PY2)_DEBS_DEPENDS = $(LIBSWSSCOMMON) $(PYTHON_SWSSCOMMON)
+$(SONIC_YCABLE_PY2)_PYTHON_VERSION = 2
+SONIC_PYTHON_WHEELS += $(SONIC_YCABLE_PY2)
```

Comment:
We are moving to PY3 only, then why PY2 support?

Reply:
fixed